backend/controller/ml_model/templates/output.sql

```diff
@@ -0,0 +1,95 @@
+CREATE OR REPLACE TABLE `{{project_id}}.{{model_dataset}}.output` AS (
+  WITH prior_table AS (
+    SELECT SPLIT(MAX(table_id), 'events_')[OFFSET(1)] AS suffix
+    FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`
+    WHERE table_id LIKE 'events_%'
+    AND table_id NOT IN (
+      SELECT MAX(table_id) FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`)
+  ),
+  events AS (
+    SELECT DISTINCT
+      {{unique_id}},
+      event_name AS name
+      event_params AS params
+    FROM `{{project_id}}.{{ga4_dataset}}.events_*`
+    WHERE _TABLE_SUFFIX = (SELECT suffix FROM prior_table)
+  ),
+  users_with_score AS (
+    SELECT DISTINCT
+      {{unique_id}}
+    FROM events, UNNEST(params) AS params
+    WHERE name = 'prop_score'
```

Comment:
This solution is actually the thing creating this parameter. This is looking to see if there are things that it's already created in GA4 via this process to avoid re-scoring users that have already been scored.